src/invidious/views/template.ecr

```diff
@@ -28,118 +29,205 @@
-        <div class="pure-u-1-4">
-          <a title="<%= translate(locale, "Subscriptions") %>" href="/feed/subscriptions" class="pure-menu-heading">
-            <% notification_count = env.get("user").as(User).notifications.size %>
-            <% if notification_count > 0 %>
-            <%= notification_count %> <i class="icon ion-ios-notifications"></i>
-            <% else %>
-            <i class="icon ion-ios-notifications-outline"></i>
-            <% end %>
+        </li>
+        <li class="pure-menu-item">
+          <a title="<%= translate(locale, " Gaming") %>" href="/feed/trending?type=Gaming" class="pure-menu-link">
+            <i class="icon ion-logo-game-controller-b"></i>
+            <%= translate(locale, "Gaming") %>
           </a>
-        </div>
-        <div class="pure-u-1-4">
-          <a title="<%= translate(locale, "Preferences") %>" href="/preferences?referer=<%= env.get?("current_page") %>" class="pure-menu-heading">
-            <i class="icon ion-ios-cog"></i>
+        </li>
+        <li class="pure-menu-item">
+          <a title="<%= translate(locale, " News") %>" href="/feed/trending?type=News" class="pure-menu-link">
+            <i class="icon ion-ios-paper"></i>
+            <%= translate(locale, "News") %>
           </a>
-        </div>
-        <div class="pure-u-1-4">
-          <a href="/signout?referer=<%= env.get?("current_page") %>&token=<%= env.get?("token") %>&challenge=<%= env.get?("challenge") %>" class="pure-menu-heading">
-            <%= translate(locale, "Sign out") %>
+        </li>
+        <li class="pure-menu-item">
+          <a title="<%= translate(locale, " Movies") %>" href="/feed/trending?type=Movies" class="pure-menu-link">
+            <i class="icon ion-ios-film"></i>
+            <%= translate(locale, "Movies") %>
           </a>
-        </div>
-        <% else %>
-        <div class="pure-u-1-3">
-          <a href="/toggle_theme?referer=<%= env.get?("current_page") %>" class="pure-menu-heading">
-            <% if env.get?("preferences").try &.as(Preferences).dark_mode %>
-            <i class="icon ion-ios-sunny"></i>
-            <% else %>
-            <i class="icon ion-ios-moon"></i>
-            <% end %>
+        </li>
+        <li class="pure-menu-item menu-item-divided">
+          <a title="<%= translate(locale, " History") %>" href="/feed/history" class="pure-menu-link">
+            <i class="icon ion-ios-timer"></i>
+            <%= translate(locale, "History") %>
           </a>
-        </div>
-        <div class="pure-u-1-3">
-          <a title="<%= translate(locale, "Preferences") %>" href="/preferences?referer=<%= env.get?("current_page") %>" class="pure-menu-heading">
-            <i class="icon ion-ios-cog"></i>
+        </li>
+        <li class="pure-menu-item menu-item-divided">
+          <a title="<%= translate(locale, " Preferences") %>" href="/preferences?referer=
+            <%= env.get?("current_page") %>" class="pure-menu-link">
```

Comment:
This should be `href="/preferences?referer=<%= env.get?("current_page") %>"`, since otherwise the given URL contains the leading spaces, so it links to `/preferences?referer=             %2Ffeed%2Fsubscriptions`. If you want, you can still spread the attributes over multiple lines, something like:
```ecr
<a title="<%= translate(locale, " Preferences") %>"
href="/preferences?referer=<%= env.get?("current_page") %>"
class="pure-menu-link">
```

Reply:
Seems like this was the Atom "beautifier" who broke this. Should be fixed now.